tests/framework/test_environment.cpp

```diff
@@ -211,16 +211,37 @@ void FrameworkEnvironment::add_explicit_layer(ManifestLayer layer_manifest, cons
 
 void FrameworkEnvironment::add_layer_impl(ManifestLayer& layer_manifest, const std::string& json_name,
                                           fs::FolderManager& folder_manager, ManifestCategory category) {
+    // We have a special case for any of the wrap objects layers so get the string to the path
+    std::string wrap_layer = fs::fixup_backslashes_in_path(fs::path(TEST_LAYER_WRAP_OBJECTS)).str();
+
+    // Strip off ending
+#if defined(WIN32)
+    size_t index = 0;
+    index = wrap_layer.find(".dll", index);
+    assert(index != std::string::npos);
+    wrap_layer.replace(index, 4, "");
```

Comment:
Just a style nit, I would create a single variable that is defined by the win32/linux/macOS.
```c++
#if defined(WIN32)
    std::string library_extension = ".dll";
#elif defined(__APPLE__)
    std::string library_extension = ".dylib";
#else
    std::string library_extension = ".so";
#endif
    size_t index = 0;
    index = wrap_layer.find(library_extension, index);
    assert(index != std::string::npos);
    wrap_layer.replace(index, library_extension.size(), "");
```